### PR TITLE
readability on project page #517

### LINKED ISSUE #517: readability on project page
Some elements like the phase description are difficult to read due very long lines of text. 

bildschirmfoto 2016-11-25 um 13 40 27

### PR DESCRIPTION
**Pull request by damusp**
narrow & medium containers on project page for better readability

fixes #517

### CODE REVIEW

File: euth_wagtail/assets/scss/components/_ideas.scss [hunk 1]
```diff
@@ -24,7 +24,7 @@
 
     p {
         font-family: $font-family-sans-serif;
-        @include rem(font-size, 16px);
+        font-size: $font-size-medium;
```
Comment (vellip) [inline euth_wagtail/assets/scss/components/_ideas.scss]: use `@include rem(font-size, $font-size-medium);` instead

File: euth/ideas/templates/euth_ideas/idea_list.html [hunk 2]
```diff
@@ -2,88 +2,90 @@
...
 {% block phase_content %}
-<div class="idea-list">
-    {% has_perm 'euth_ideas.propose_idea' request.user view.module as add_button_enabled %}
-    {% if add_button_enabled %}
-    <a href="{% url 'idea-create' slug=view.module.slug %}" class="idea-list-button" id="idea-create">
-        <i class="fa fa-plus idea-list-button-icon" aria-hidden="true"></i>
-        {% trans 'Create Proposal' %}
-    </a>
-    {% endif %}
+<div class="container-medium">
```
Comment (vellip) [inline euth/ideas/templates/euth_ideas/idea_list.html]: there a reason you use `.container-medium` here and `.container-narrow` in the other file?

Comment (slomo) [inline euth/ideas/templates/euth_ideas/idea_list.html]: @damusp Have you clear things up her with @vellip?

Comment (damusp) [inline euth/ideas/templates/euth_ideas/idea_list.html]: @slomo yes, this matter has been cleared. I've used the different classes fully intentional.

**Pull request merged**